artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/connect/mirror/AMQPMirrorControllerSource.java

```diff
@@ -89,9 +90,15 @@ public class AMQPMirrorControllerSource extends BasicMirrorController<Sender> im
    // Capabilities
    public static final Symbol MIRROR_CAPABILITY = Symbol.getSymbol("amq.mirror");
    public static final Symbol QPID_DISPATCH_WAYPOINT_CAPABILITY = Symbol.valueOf("qd.waypoint");
+   public static final Symbol NO_FORWARD = Symbol.getSymbol("amq.no.forward");
+   public static final Symbol NO_FORWARD_SOURCE = Symbol.getSymbol("amq.no.forward.source");
+   public static final Symbol RECEIVER_ID_FILTER = Symbol.getSymbol("amq.receiver.id.filter");
```

Comment:
NO_FORWARD_SOURCE and RECEIVER_ID_FILTER arent being used as a capability, so shouldn't be in this group of constants that are for capabilities. They also likely don't need to be of Symbol type as a result.

Reply:
The NO_FORWARD_SOURCE is used as a BrokerProperty which is and that requires to have a Symbol passed as a parameter:

![image](https://github.com/user-attachments/assets/990bb32b-9af7-4c85-99fc-492fe7abe423)